code/modules/cargo/packs.dm

```diff
@@ -601,7 +601,6 @@
 					/obj/item/clothing/mask/russian_balaclava,
 					/obj/item/clothing/head/helmet/rus_ushanka,
 					/obj/item/clothing/suit/armor/vest/russian_coat,
-					/obj/item/gun/ballistic/rifle/boltaction,
```

Comment:
we really shouldn't be balancing things by making them more RNG-dependent. this just turns russian surplus crates into even more of a lottery ("did I just waste 10 crates' worth of credits on ammo that I can't use, or did I actually roll the one good thing on this loot table that I actually wanted to buy?")

Reply:
It would feel more like gatcha than it did before, 4k gun crate with random ballistics 1% for bullpop shotgun 😄 
but yeah, I might go back to old chances + add the cleaning kit